cmd/observe/observe.go

```diff
@@ -177,14 +180,53 @@ more.`,
 
 	observerCmd.Flags().Var(filterVar(
 		"verdict", ofilter,
-		fmt.Sprintf("Show only flows with this verdict [%v, %v]", pb.Verdict_FORWARDED, pb.Verdict_DROPPED)))
+		fmt.Sprintf(
+			"Show only flows with this verdict [%v, %v, %v]",
+			pb.Verdict_FORWARDED, pb.Verdict_DROPPED, pb.Verdict_ERROR),
+	))
+	observerCmd.RegisterFlagCompletionFunc("verdict", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		return []string{
+			pb.Verdict_FORWARDED.String(),
+			pb.Verdict_DROPPED.String(),
+			pb.Verdict_ERROR.String(),
+		}, cobra.ShellCompDirectiveDefault
+	})
 
 	observerCmd.Flags().Var(filterVar(
 		"http-status", ofilter,
 		`Show only flows which match this HTTP status code prefix (e.g. "404", "5+")`))
+	observerCmd.RegisterFlagCompletionFunc("http-status", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		httpStatus := []string{
+			"100", "101", "102", "103",
+			"200", "201", "202", "203", "204", "205", "206", "207", "208",
+			"226",
+			"300", "301", "302", "303", "304", "305", "307", "308",
+			"400", "401", "402", "403", "404", "405", "406", "407", "408", "409",
+			"410", "411", "412", "413", "414", "415", "416", "417", "418",
+			"421", "422", "423", "424", "425", "426", "428", "429",
+			"431",
+			"451",
+			"500", "501", "502", "503", "504", "505", "506", "507", "508",
+			"510", "511",
+		}
+		return httpStatus, cobra.ShellCompDirectiveDefault
+	})
 	observerCmd.Flags().Var(filterVar(
 		"http-method", ofilter,
 		`Show only flows which match this HTTP method (e.g. "get", "post")`))
+	observerCmd.RegisterFlagCompletionFunc("http-method", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		return []string{
+			"connect",
+			"delete",
+			"get",
+			"head",
+			"options",
+			"patch",
+			"post",
+			"put",
+			"trace",
```

Comment:
nit: HTTP methods are usually seen in all-caps (e.g. "POST" rather "post"). Since `filterByHTTPMethods` does a case insensitive check it wouldn't functionally change anything.